elasticdl/python/worker/worker.py

```diff
@@ -202,6 +188,16 @@ def _init_from_args(self, args):
             checkpoint_path=args.checkpoint_dir,
         )
 
+        self._allreduce_trainer = None
+        if (
+            self._distribution_strategy == DistributionStrategy.ALLREDUCE
+            and args.num_workers > 1
```

Comment:
args.num_workers can be 1. Thus, remove this restriction.

Reply:
Yeah!